docs(root): add smc tss examples (#3736)
BitGo/BitGoJS: examples/ts/tss-smc/HOWTO.md

# HOW TO

## Create an EdDSA TSS Self-Managed Cold Wallet

> This guide explains the steps required to create an EdDSA TSS self-managed cold wallet using the provided repository.

Reviewer: ```suggestion
This guide explains how to create a self-managed TSS cold wallet for EdDSA assets using the provided repository.
```

## Prerequisites

> Before starting the process, ensure the following prerequisites are met:
> 

Reviewer: ```suggestion
```

> - Have two OVCs (Online Voting Centers) instances running, one for the User and one for the Backup.

Reviewer: ```suggestion
- Set up 2 air-gapped machines with the BitGo Offline Vault Console (OVC) (1 for the user key and 1 for the backup key).
```

> - Node.js and npm (Node Package Manager) should be installed locally.

Reviewer: ```suggestion
- Install Node.js
- Install Node Package Manager (npm)
```

> - Run `npm install` from the root directory of the repository to install the necessary dependencies.

Reviewer: ```suggestion
- Install the necessary dependencies by running `npm install` from the root directory of the repository.
```

## Steps

> Follow these steps:
> 

Reviewer: ```suggestion
```

> 1. Open OVC 1 and select `Generate TSS Key Share` from the menu. Choose OVC 1 as the instance.

Reviewer: ```suggestion
1. Open OVC on one of the air-gapped machines. Select `Generate TSS Key Share` from the menu. Select `OVC 1` as the instance.
```

> 2. Open OVC 2 and select `Generate TSS Key Share` from the menu. Choose OVC 2 as the instance.

Reviewer: ```suggestion
2. Open the OVC on the other the air-gapped machine. Select `Generate TSS Key Share` from the menu. Select `OVC 2` as the instance.
```

> **Do not close the OVCs after starting the process**

Reviewer: ```suggestion
**Note:** Keep both instances of the OVC open until you finish creating the wallet.
```

> 3. In OVC 1, follow the instructions and choose the "Single Private Key" option. Download the file required for OVC 2.

Reviewer: ```suggestion
3. In OVC 1, select the `Single Private Key` option and download the JSON.
```

> 4. In OVC 2, upload the file received from OVC 1 and choose the "Single Private Key" option. Download the json file required for the step 5.

Reviewer: ```suggestion
4. In OVC 2, upload the JSON and select the `Single Private Key` option. Download the updated JSON file.
```

> 5. Open the script `create-smc-wallet-step-1` and follow the instructions. Execute it using `ts-node create-smc-wallet-step-1.ts`. This will generate a file named `toOVC1-step-1-timestamp.json` (required for step 6). Additionally, note down the Bitgo Key Id displayed in the console.

Reviewer: ```suggestion
5. Open the script `create-smc-wallet-step-1` and follow the instructions. Execute it using `ts-node create-smc-wallet-step-1.ts`. This generates `toOVC1-step-1-timestamp.json`. Make note of the BitGo key ID displayed in the console.
```

> 6. Go back to OVC 1, upload the JSON file generated in the previous step (5), and proceed with the process. Download the JSON file required for OVC 2 (needed for step 7). Also, download the private key file (**store it securely!**) and the public key file (required for step 8).

Reviewer: ```suggestion
6. In OVC 1, upload the JSON file generated in the previous step. Download the updated JSON, the private-key file, and the public-key file.
**Note:** Store the private-key file in a secure location.
```

> 7. Go back to OVC 2, upload the JSON file generated in step 6, and continue the process. Download the private key file (**store it securely!**) and the public key file (required for step 8).

Reviewer: ```suggestion
7. In OVC 2, upload the JSON generated previous step. Download the private-key file and the public-key file.
**Note:** Store the private-key file in a secure location.
```

> 8. Open the script `create-smc-wallet-step-2` and follow the instructions. Ensure that you include the public key files from OVC 1 and OVC 2. Additionally, provide the Bitgo Key Id obtained in step 5. Execute the script using `ts-node create-smc-wallet-step-2.ts`. This will create a wallet, and the relevant information will be stored in a file named `wallet-walletName.json` (for informational purposes only).

Reviewer: ```suggestion
8. Open the script `create-smc-wallet-step-2` and follow the instructions. Include the public-key files from OVC 1 and OVC 2 and the BitGo key ID from step 5. Execute the script using `ts-node create-smc-wallet-step-2.ts`. This creates the wallet, and stores the relevant information in `wallet-walletName.json` (for informational purposes only).
```
